.env

```
@@ -382,28 +376,28 @@ VORTEX_NOTIFY_WEBHOOK_URL=
 # Used to showcase Vortex without asking users to perform additional steps.    #
 #                                                                              #
-# Remove this section after completing database download integration.          #
+# Remove this section after completing database fetch integration.          #
 #                                                                              #
 ################################################################################
 
-#;< DB_DOWNLOAD_SOURCE_URL
-# URL of the database used for demonstration with URL database download type.
+#;< DB_FETCH_SOURCE_URL
+# URL of the database used for demonstration with URL database fetch type.
 VORTEX_FETCH_DB_URL=https://github.com/drevops/vortex/releases/download/1.37.0/db.demo.sql
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The VORTEX_FETCH_DB_URL key is duplicated

(DuplicatedKey)